src/hastings_util.erl

```diff
@@ -147,6 +147,7 @@ maybe_create_local_purge_doc(Db, Index) ->
         {not_found, _Reason} ->
             DefaultPurgeSeq = couch_db:get_purge_seq(Db),
             PurgeSeq = try easton_index:get(Index#h_idx.pid, purge_seq) of
+                false -> DefaultPurgeSeq;
```

Comment:
can it be possible that the index `purge_seq` exists, but something happened with easton and it returns `false`, and we return DB purge_seq which is different than index purge_seq?

Reply:
Its always possible that Easton could like about its purge_seq but that'd be a bug in Easton and not this code relying on it, I think?

Reply:
@davisp @jiangphcn : so iiuc, `false` in this case means that there is no existing value for `purge_seq` for this particular index. Unfortunately, I don't understand purge well enough to know how `PurgeSeq` will be used later on. So what is the importance of assigning it to default db purge_seq value vs say 0?

Reply:
understood, thanks for clarifying